std/bitmanip.d

```diff
@@ -300,7 +300,7 @@ The example above creates a tagged reference to an Object in the struct A. This
 as $(D taggedPointer), except the first argument which must be a class type instead of a pointer type.
 
 $(RED Warning: Don't use $(D taggedClassRef) with references to garbage collected objects, as it will result in
-undefined behaviour. See $(LINK http://dlang.org/garbage.html) for details.)
+undefined behaviour. See $(LINK ../spec/garbage.html) for details.)
```

Comment:
Or even better, define `ROOT_DIR` to be `/` for DDox, or implement it in DDox/dpl-docs to be a relative link to the root directory :)